mailbox/cassandra/pom.xml

```diff
@@ -62,6 +62,10 @@
             <type>test-jar</type>
```

Comment:
Hm. IMO we should have separate maven projects for blobs. I don't like this dependency between mailbox and data.

Maybe ObjectStore and BlobIds can be a top-level backend api?

Reply:
Yes, blob management desevers its own project